README.md

```diff
@@ -33,14 +33,8 @@ Requirements
   - flask-bootstrap
   - flask-nav
 
-On Ubuntu 18.10 as reference distro, those dependencies can easily installed
-with:
-```
-# apt install python3-sklearn python3-git python3-pygit2 python3-fuzzywuzzy
-	      python3-flaskext.wtf python3-pip python3-tqdm \
-              git procmail
-$ pip3 install --user flask-bootstrap flask-nav anytree
-```
+On Ubuntu 19.04 as reference distro, you can refer to this [Dockerfile](https://github.com/lfd/PaStA/blob/master/docker/pasta-skeleton.dockerfile)
+to download the required dependencies.
```

Comment:
Can we reword it a bit and remove the "you". "Please refer to ...". For the link, can we somehow use markdown-magic to reference relatively without providing the full github link? I'd like to preserve a maximum line length of 80 characters. Thanks!